index.fr.md

```diff
@@ -0,0 +1,147 @@
+-   **les règles WAI** ([WCAG, ATAG, UAAG décrites ci-dessous](#guidelines)) définissent comment mettre en place les équivalents textuels dans la perspective de l’accessibilité pour les différentes composantes&nbsp;;
+-   les **créateurs** fournissent la formulation appropriée de l’équivalent textuel&nbsp;;
+-   les **outils de création** permettent, facilitent et promeuvent la fourniture d’un équivalent textuel dans une page web&nbsp;;
+-   les **outils d’évaluation** sont utilisés pour faciliter la vérification de la présence d’équivalents textuels&nbsp;;
+-   les **agents utilisateurs** fournissent des interfaces à destination des êtres humains et des machines pour l’équivalent textuel&nbsp;;
+-   les **technologies d’assistance** fournissent un accès aux êtres humains à l’équivalent textuel suivant différentes modalités&nbsp;;
+-   les **utilisateurs** savent comment obtenir l’équivalent textuel depuis leur agent utilisateur ou leur technologie d’assistance ou les deux, selon que de besoin.
+ 
+### Le cycle de mise en œuvre
+ 
+Quand les fonctionnalités d’accessibilité sont effectivement mises en œuvre dans une composante, les autres composantes sont plus susceptibles de les mettre en œuvre elles aussi.
+ 
+{% assign example_url = "/fundamentals/components/examples/#cycle" | relative_url %}
+![illustration du cycle de mise en œuvre, description détaillée à {{ example_url }}]({{ "/content-images/wai-components/cycle.fr.png" | relative_url }}){:longdesc="{{example_url}}"}
+ 
+- Quand **les navigateurs web, les lecteurs de média, les technologies d’assistance, et les autres agents utilisateurs** fournissent une fonctionnalité d’accessibilité, les utilisateurs sont plus susceptibles de l’exiger  et les créateurs sont plus susceptibles de l’implémenter dans leur **contenu**&nbsp;;
+- Quand les créateurs veulent implémenter une fonctionnalité d’accessibilité dans leur **contenu**, ils sont plus susceptibles d’exiger que leur **outil de création** en rende simple la mise en œuvre&nbsp;;
+- Quand **les outils de création** rendent simple la mise en œuvre d’une fonctionnalité, les créateurs sont plus susceptibles de la mettre en œuvre dans leur **contenu**&nbsp;;
+- Quand une fonctionnalité d’accessibilité est mise en œuvre dans  **la plupart des contenus**, les créateurs et les utilisateurs sont plus susceptibles d’exiger que les **agents utilisateurs** la fournissent.
+ 
+### Quand une composante est déficiente
+ 
+Si une fonctionnalité d’accessibilité n’est pas mise en œuvre dans une composante, il y a peu de motivation pour les autres composantes pour l’implémenter s’il n’en résulte pas une expérience utilisateur accessible. Par exemple les créateurs n’implémenteront pas une fonctionnalité d’accessibilité que les outils de création ne fournissent pas et que la plupart des navigateurs et des technologies d’assistance ne mettent pas en œuvre systématiquement.
```

Comment:
"n'implémenteront pas" seems a bit strong to translate "are unlikely to implement". What do you think of this suggestion?

```suggestion
Si une fonctionnalité d’accessibilité n’est pas mise en œuvre dans une composante, il y a peu de motivation pour les autres composantes pour l’implémenter s’il n’en résulte pas une expérience utilisateur accessible. Par exemple il est peu probable que les créateurs implémentent une fonctionnalité d’accessibilité que les outils de création ne fournissent pas et que la plupart des navigateurs et des technologies d’assistance ne mettent pas en œuvre systématiquement.
```

Reply:
n’implémenteront probablement pas ?